container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2454,7 +2454,10 @@ configuration:
       secret-generator:
         roles: [configgin-role, secrets-role]
         cluster-roles: [nonprivileged]
-      tests:
+      tests-brain:
+        roles: [configgin-role, test-role]
```

Comment:
This still generates `test-role` as part of the chart, rather as part of the task definition.

Reply:
IOW the `test-role` must be split into `test-brain-role` and `test-sits-role` as well, right ?

Reply:
Right!

Reply:
SITS don't need its own Role. The test ClusterRole is sufficient for it to be able to create a port-forward. I removed it and renamed `test-role` to `test-role-brain` so it is clearer that it shouldn't be reused.